ftw/upgrade/tests/test_upgrade_step.py

```diff
@@ -87,7 +87,8 @@ class Step(UpgradeStep):
             def __call__(self):
                 for obj in self.objects({'portal_type': 'Folder'},
                                         'Log message',
-                                        logger=testcase.logger):
+                                        logger=testcase.logger,
+                                        savepoints=False):
```

Comment:
I think the docstring for `self.objects` should also point out that the default (`savepoints=None`) **will** create savepoints - that's somewhat non-obvious. 

The docs for `self.objects` should probably also point that out: https://github.com/4teamwork/ftw.upgrade#id11

Reply:
@lukasgraf I have update the documentation of `self.objects` in the readme; could you review it again?